### DISCUSSION TITLE
cURL error 60 when accessing /cp

### DISCUSSION THREAD

**Discussion by jmannmaqqie** [statamic/cms#8471] [Q&A] [answered]
After redeploying our website the control panel no longer works, the site itself is fine though.

We are getting the following message: `cURL error 60: SSL: no alternative certificate subject name matches target host name 'outpost.statamic.com' (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://outpost.statamic.com/v3/query`

Any help in resolving this would be greatly appreciated!

**Comment by jmannmaqqie** [comment, marked as answer]
Turns out this was a problem with the [latest release of cURL](https://askubuntu.com/questions/1478616/ssl-no-alternative-certificate-subject-name-matches-new-release-of-curlusn-6). Fixed now by moving to 7.81.0-1ubuntu1.13

**Reply by jesseleite** [reply]
We also [improved our error handling a bit in core](https://github.com/statamic/cms/pull/8475) for this, but definitely a cURL issue 👍
